rust/xaynet-server/src/settings.rs

```diff
@@ -239,8 +239,8 @@ pub struct PetSettings {
 impl Default for PetSettings {
     fn default() -> Self {
         Self {
-            min_sum_count: 1_usize,
-            min_update_count: 3_usize,
+            min_sum_count: 1_u64,
```

Comment:
Do we actually need the _usize and _u64 suffixes here? If not maybe we could use this cleanup PR for removing them.